src/pybind/mgr/dashboard/controllers/rbd_mirroring.py

```diff
@@ -651,7 +651,9 @@ def status(self):
             return status
         if not CephService.get_service_list('rbd-mirror') and not CephService.get_pool_list('rbd'):
             status['available'] = False
-            status['message'] = 'RBD mirroring is not configured'  # type: ignore
+            status['message'] = 'No default "rbd" Pool or "rbd-mirror" Service in the ' \
```

Comment:
do we need to capitalize Pool and Service?
```suggestion
            status['message'] = 'No default "rbd" pool or "rbd-mirror" service in the ' \
```